apps/samradsgatt/codegen.yml

```diff
@@ -0,0 +1,19 @@
+schema:
+  - apps/api/src/api.graphql
+documents: apps/samradsgatt/**/*.{ts,tsx}
+generates:
+  apps/samradsgatt/graphql/schema.ts:
+    plugins:
+      - typescript
+      - typescript-operations
+      - typescript-react-apollo
+    config:
+      exportFragmentSpreadSubTypes: true
+      scalars:
+        DateTime: Date
+        JSON: '{ [key: string]: any }'
+      namingConvention:
+        typeNames: change-case#pascalCase
```

Comment:
This generates a humongous schema.ts file with all of the types. Should use [`near-operation-file` preset](https://the-guild.dev/graphql/codegen/plugins/presets/near-operation-file-preset) (or [import-types](https://the-guild.dev/graphql/codegen/plugins/presets/import-types-preset)). 

Check out [this config](https://github.com/island-is/island.is/blob/42c53faa405dd8af0ee0a79243e8977eafa4e8dd/libs/portals/admin/air-discount-scheme/codegen.yml) where we make it generate multiple "generated.ts" files which all share types from "@island.is/api/schema".